doc/howto/cluster_config_networks.md

```diff
@@ -1,60 +1,103 @@
-```
+       lxc network create --target server1 my-network
+       lxc network create --target server2 my-network
+       lxc network create --target server3 my-network
 
-At this point the network hasn't been actually created yet, but just defined
-(it's state is marked as Pending if you run `lxc network list`).
+   ```{note}
+   You can pass only the member-specific configuration keys `bridge.external_interfaces`, `parent`, `bgp.ipv4.nexthop` and `bgp.ipv6.nexthop`.
+   Passing other configuration keys results in an error.
+   ```
 
-Now run:
+   These commands define the network, but they don't create it.
+   If you run `lxc network list`, you can see that the network is marked as "pending".
+1. Run the following command to instantiate the network on all cluster members:
 
-```bash
-lxc network create my-network
-```
+       lxc network create my-network
 
-The network will be instantiated on all nodes. If you didn't define it on a particular node, or a node is down,
-an error will be returned.
+   ```{note}
+   You can add configuration keys that are not member-specific to this command.
+   ```
 
-You can pass to this final `network create` command any configuration key which is not node-specific (see above).
+   If you missed a cluster member when defining the network, or if a cluster member is down, you get an error.
+
+Also see {ref}`network-create-cluster`.
 
 ## Separate REST API and clustering networks
 
-You can configure different networks for the REST API endpoint of your clients
-and for internal traffic between the nodes of your cluster (for example in order
-to use a virtual address for your REST API, with DNS round robin).
+You can configure different networks for the REST API endpoint of your clients and for internal traffic between the members of your cluster.
+This separation can be useful, for example, to use a virtual address for your REST API, with DNS round robin.
+
+To do so, you must specify different addresses for [`core.https_address`](server) (the address for the REST API) and [`cluster.https_address`](server) (the address for internal cluster traffic).
+
+1. Configure the bootstrap server:
+
+   `````{tabs}
+
+   ````{group-tab} Interactive configuration
+   When running `lxd init`, specify the address for the REST API when asked for the IP address or DNS name for the member (for example, `my.lxd.cluster:8443`).
+   This value is used for the `core.https_address` configuration.
+
+   After the initialization, set the `cluster.https_address` configuration to the address for internal cluster traffic.
+   For example:
+
+       lxc config set cluster.https_address 192.0.2.171:8443
+
+   ````
+   ````{group-tab} Preseed file
+   Specify the address for the REST API for the `core.https_address` configuration and the address for internal cluster traffic for the `cluster.https_address` configuration.
+
+   For example:
+
+   ```yaml
+   config:
+     core.https_address: my.lxd.cluster:8443
```

Comment:
Maybe for demonstration purposes use a wildcard address, such as 0.0.0.0:8443 to show that the core.https_address can listen on multiple interfaces but the cluster.https_address cannot.